components/ArticlePageLayout.js

```diff
@@ -345,31 +377,40 @@ function ArticlePageLayout({
         <h1>{jt`Messages reported by user that reported “${searchedUserArticleElem}”`}</h1>
       )}
 
-      <Grid container alignItems="center" justify="space-between" spacing={2}>
-        <Grid item xs={12} lg="auto">
-          <Typography variant="h4">{title}</Typography>
-        </Grid>
-        <Grid item xs={12} lg="auto">
+      <Box
+        display="flex"
+        alignItems="center"
+        justifyContent={{ xs: 'center', md: 'space-between' }}
+        flexDirection={{ xs: 'column', md: 'row' }}
+        mb={2}
+      >
+        <Typography variant="h4">{title}</Typography>
+        <Box my={1}>
           <FeedDisplay
             feedUrl={`${PUBLIC_URL}/api/articles/rss2?${queryString}`}
           />
-        </Grid>
-      </Grid>
+        </Box>
+      </Box>
 
-      <Grid container alignItems="center" justify="space-between">
-        <Grid item>
+      <Box
+        display="flex"
+        alignItems="center"
+        justifyContent="space-between"
+        overflow="scroll"
```

Comment:
![截圖 2020-05-16 下午7 56 13](https://user-images.githubusercontent.com/12843409/82119267-913d3500-97af-11ea-917e-db1faae0bc0d.png)
the `overflow: hidden `is because that when selecting custom time range, the total length would be too long that breaks the layout

Reply:
Hmm I didn't think of that. I think we can have a wrapping flexbox here, allowing the sort control go to second line.